Mathlib/Analysis/Calculus/BumpFunction/Basic.lean

```diff
@@ -66,6 +66,7 @@ The structure `ContDiffBump` contains the data required to construct the functio
 real numbers `rIn`, `rOut`, and proofs of `0 < rIn < rOut`. The function itself is available through
 `CoeFun` when the space is nice enough, i.e., satisfies the `HasContDiffBump` typeclass. -/
 structure ContDiffBump (c : E) where
+  /-- real numbers `0 < rIn < rOut` -/
```

Comment:
Does it assign the same docstring to both `rIn` and `rOut`?

Reply:
Yes, it does. Given these fields are related, this actually felt nicer.
If you prefer the following phrasing, feel free to apply it directly (and manually delete the next line).
```suggestion
/-- real number -/
rIn : ℝ
/-- real number, larger than rIn -/
rOut : ℝ

Reply:
No, mentioning both LGTM.